### ISSUE TITLE
.getPath - Kerning Support

### ISSUE THREAD

**Issue by villanus** [nodebox/opentype.js#187]
We constructing a path from a loaded font how do you control Letter Spacing (or Kerning) so that we can control the spacing between each letter?

**Comment by Jolg42** [comment]
Hi @villanus 
In fact there the writing part is not implemented yet for the kerning so you can only read it.

**Comment by villanus** [comment]
got it.  it would be great to have kerning and alignment support.

**Comment by fdb** [comment]
I agree. The simple `kern` table, which does simple horizontal adjustment, is easy to implement. The OpenType `GPOS` table is more full-featured but also a lot more complex.

**Comment by Jolg42** [comment]
As `GSUB` is now implemented (Thank you @fpirsch!) it would be a lot easier to implement `GPOS` because they have a base in common. So if anyone is interested... 😉

**Comment by fpirsch** [comment]
I could do it if only I had some time...
